extensions/parquet/base/src/main/java/io/deephaven/parquet/base/BigDecimalParquetBytesCodec.java

```diff
@@ -28,43 +28,46 @@ public class BigDecimalParquetBytesCodec implements ObjectCodec<BigDecimal> {
     private final byte[] nullBytes;
 
     /**
-     * Verify that the scale and precision are valid.
+     * Verify that the precision and scale are valid.
      *
-     * @throws IllegalArgumentException if the provided scale and/or precision is invalid
+     * @throws IllegalArgumentException if the provided precision and/or scale is invalid
      */
-    public static void verifyScaleAndPrecision(final int scale, final int precision) {
+    public static void verifyPrecisionAndScale(final int precision, final int scale) {
```

Comment:
I did the reordering to make it consistent with everywhere else.
Since precision and scale are both ints, I am trying to be careful in how we pass these around.